docs/en/engines/database-engines/materialize-mysql.md

```diff
@@ -5,9 +5,9 @@ toc_title: MaterializeMySQL
 
 # MaterializeMySQL {#materialize-mysql}
 
- Creates ClickHouse database with all the tables existing in MySQL, and all the data in those tables. 
+Creates ClickHouse database with all the tables existing in MySQL, and all the data in those tables.
```

Comment:
Should mention that this is experimental feature.
Just

"This feature is experimental."

without any huge warnings, etc...